src/plugin.js

```diff
@@ -233,17 +232,7 @@ class PlaylistMenu extends Component {
   }
 
   createEl() {
-    const settings = this.options_;
-
-    if (settings.el) {
-      return settings.el;
-    }
-
-    const ol = document.createElement('ol');
-
-    ol.className = settings.className;
-    settings.el = ol;
-    return ol;
+    return dom.createEl('div', {className: this.options_.className});
```

Comment:
isn't this a breaking change?

Reply:
Not really because this shouldn't have been returning an `ol` to begin with. The `createPlaylist_` method creates a `ol` and sticks it into `this.el_`, meaning you have `<ol><ol></ol></ol>`, which is invalid HTML.

Reply:
Yeah, sounds reasonable. Just wanted to be extra sure. If this does end up breaking someone for some reason, we could just revert in a patch and make a new major.